.github/workflows/ci.yml

```diff
@@ -57,6 +57,10 @@ jobs:
 
       - run: "ci/build.sh"
 
+      - name: "(Linux only) Build the entire package set"
+        if: "${{ runner.os == 'Linux' }}"
+        run: "stack exec ci/build-package-set.sh"
```

Comment:
Does this need to be `stack exec bash ci/build-package-set.sh` or something like that?